openshift/templates/filesystem-provider.bc.json

```diff
@@ -0,0 +1,136 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "mds-filesystem-provider-bc",
+    "creationTimestamp": null
+  },
+  "objects": [
+    {
+      "kind": "ImageStream",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "${NAME}"
+      }
+    },
+    {
+      "kind": "BuildConfig",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "${NAME}${SUFFIX}",
+        "labels": {
+          "app": "${NAME}${SUFFIX}"
+        }
+      },
+      "spec": {
+        "triggers": [
+		{
+			"type": "ConfigChange"
+		},
+		{
+			"type": "ImageChange"
+		},
+        "runPolicy": "Serial",
+        "source": {
+          "type": "Git",
+          "git": {
+            "uri": "${SOURCE_REPOSITORY_URL}",
+            "ref": "${SOURCE_REPOSITORY_REF}"
+          },
+          "contextDir": "${SOURCE_CONTEXT_DIR}"
+        },
+        "strategy": {
+          "type": "Source",
+          "sourceStrategy": {
+            "from": {
+              "kind": "${SOURCE_IMAGE_KIND}",
+              "name": "${SOURCE_IMAGE_NAME}:${SOURCE_IMAGE_TAG}"
+            },
+			"env": [
+              {
+                "name": "DOTNET_STARTUP_PROJECT",
+                "value": "${DOTNET_STARTUP_PROJECT}"
+              }
+            ]
+          }
+        },
+        "output": {
+          "to": {
+            "kind": "ImageStreamTag",
+            "name": "${NAME}:${VERSION}"
+          }
+        },
+        "completionDeadlineSeconds": 600
+        ]
+      }
+    }
+  ],
+  "parameters": [
+    {filesystem-provider
```

Comment:
I think this can be removed?

Reply:
yeah for some reason it was yellow on white in notepad++ and I didn't notice it